src/shared/selectors/flags.ts

```diff
@@ -50,7 +49,6 @@ export const OSS_FLAGS = {
 
 export const CLOUD_FLAGS = {
   cloudBilling: CLOUD_BILLING_VISIBLE, // should be visible in dev and acceptance, but not in cloud
-  cursorAtEOF: false,
```

Comment:
just fyi it looks like the flag is still in configcat/idpe! So don't forget to remove it from there

Reply:
Oh seriously? Yup, will remove that too